test/unit/init/cluster/SingleThreaded.test.ts

```diff
@@ -0,0 +1,61 @@
+import { ComponentsManager } from 'componentsjs';
+import type { Resource } from '../../../../src';
```

Comment:
Resource is a Components.js type right? How is it being exported from our index?

Reply:
🤔 that could very wel be the problem. I'll look into it next week

Reply:
It didn't solve the issue, still need that `jsonld-context-parser/lib/ContextParser` mock. But I fixed the import of course.